### ISSUE TITLE
Support for old style figures

### ISSUE THREAD

**Issue by roelandschoukens** [google/fonts#1780]
It has been well-documented that all fonts shall have lining numerals by default.

However for some fonts the old style numerals would be a better fit. Case in point are the various Garamond style typefaces. (Cormorant is actually served with old style numerals).

You can do `font-variant-numeric: oldstyle-nums;` in CSS, is it possible to support features like this in Google Fonts?

**Comment by HM100** [comment]
> It has been well-documented that all fonts shall have lining numerals by default.
> 
> However for some fonts the old style numerals would be a better fit. Case in point are the various Garamond style typefaces. (Cormorant is actually served with old style numerals).
> 
> You can do `font-variant-numeric: oldstyle-nums;` in CSS, is it possible to support features like this in Google Fonts?

Widely supported except in Edge

**Comment by roelandschoukens** [comment]
Yes indeed, but you need a font which actually contains old style numerals. Google Fonts at present doesn't include them even if the original font does.
